packages/core/src/Stores/notification-store.js

```diff
@@ -369,6 +374,17 @@ export default class NotificationStore extends BaseStore {
                 this.addNotificationMessage(this.client_notifications.close_mx_mlt_account);
             }
 
+            if (
+                is_financial_assessment_incomplete &&
+                is_fully_authenticated &&
+                content_flag === ContentFlag.LOW_RISK_CR_NON_EU &&
+                is_svg
```

Comment:
Do we really need is_svg? if im not mistaken all the non-eu users are eligible for svg licenses.